index.html

```diff
@@ -0,0 +1,243 @@
+        <div class="row bg-transparent">
+            <div class="col-4 bg-transparent ps-5">
+                <a href="index.html" class="bg-transparent"><img src="photo/logo.png" id="photo" class="img-fluid h-100 padLeft" alt="logo"></a>
+            </div>
+            <div class="col-8 bg-transparent navPad">
+                <div class="d-flex flex-row-reverse bd-highlight pe-5 bg-transparent me-5">
+                    <a class=" bd-highlight bg-transparent rougesimple text-decoration-none borderRouge pt-1 pb-1 mt-3 ps-3 pe-3 margA">BICC compagn</a> 
+                    <a class="ps-2  bd-highlight bg-transparent noirfont text-decoration-none margA">News</a>
+                    <a class="ps-2  bd-highlight  bg-transparent noirfont text-decoration-none margA">Sponsor</a>
+                    <a class="ps-2  bd-highlight bg-transparent noirfont text-decoration-none margA">Join</a>
+                    <a class="ps-2  bd-highlight  bg-transparent noirfont text-decoration-none margA">Program</a>
+                    <a class="ps-2  rougesimple bd-highlight bg-transparent text-decoration-none margA" href="about.html">About</a>         
+                </div>
+            </div>
+        </div>
+    </nav>
+    <nav class="navbar navbar-expand-sm noirfont bg-transparent w-100 navbar-white" id="fixM">
+        <div class="container-fluid">
+            <button class="navbar-toggler mb-2" type="button" data-bs-toggle="collapse" data-bs-target="#collapsibleNavbar" >
+                <span class="navbar-toggler-icon" id="menu"></span>
+                <span class="fs-1 noVisible" id="cross">&cross;</span>
+              </button>
+          <!--<a class="navbar-brand noVisible fs-1" href="#" id="cross">&cross;</a>-->
+          <div class="collapse navbar-collapse" id="collapsibleNavbar">
+            <ul class="navbar-nav" id="descendant">
+                <li class="nav-item" id="homeVer" data-bs-toggle="collapse" data-bs-target="#collapsibleNavbar">
+                  <a class="nav-link" href="index.html">Home</a>
+                </li>
+                
+                <li class="nav-item" id="aboutVer" >
+                  <a class="nav-link" href="about.html">About</a>
+                </li>    
+              </ul>
+            
+          </div>
+        </div>
+    </nav>
+  </header>
+  <main>
+    <div class="accueil" id="home">
+      <h2 class="pt-5 ms-3 rouge margeB">"Hello everone, welcome to :"<br></h2>
+      <h1 class="ms-3 mb-4 rouge display-4 margeB">Open house BICC 2022</h1>
+      <div class="pWidth">
+        <p class="text-start p-3 me-3 border border-white border-2 ms-3">
+            On this wonderful day, dear guests, you will have the opportunity to discover all our works with explanation
+          </p>
+      </div>
+      <div class="margeB">
+        <p class="h2 ms-3">2023.10.15(THU) ~ 16(FRI)</p>
+        <p class="mb-5 ms-3 me-3">@Architectural design, sizing and study and more</p>
+      
```

Comment:
- [ ] `required`
- [ ] Amazing job so far! 👏 
- [ ] however, there few things you need to address
- [ ] The welcome headers and need to be moved under the logo.
- [ ] Also please try to make the `Oen house bicc` bigger like the template.
- [ ] Plese see the template below

`your template`
![image](https://user-images.githubusercontent.com/66230306/217948541-2d7d6dad-8a65-4cdc-89c2-80802aa985d4.png)


`original design`
![image](https://user-images.githubusercontent.com/66230306/217947880-52036088-4bb2-4616-9e43-ce2c1d097063.png)